.github/workflows/tests.yml

```diff
@@ -26,6 +32,17 @@ jobs:
         with:
           go-version-file: "${{ env.FTS_PATH }}/go.mod"
 
+      - name: Replace FSC dep
+        working-directory: ${{ env.FTS_PATH }}
+        run: |
+          if [ -n "${{ github.event.inputs.fsc-version }}" ]; then
+            echo "Replace FSC dependency"
+            go mod edit -replace=github.com/hyperledger-labs/fabric-smart-client=${{ github.event.inputs.fsc-version }}
```

Comment:
Maybe it would make sense to also make `hyper-ledger-labs` configurable, so that we can test the integration while its on our fork still and not when it reaches the master